src/labone/core/kernel_session.py

```diff
@@ -100,6 +100,7 @@ async def create(
         kernel_info: KernelInfo,
         server_info: ServerInfo,
         context: ZIContext | None = None,
+        timeout: float = 5,
```

Comment:
no it does not ... I changed the timeout to be ms and an integer to avoid this confusion ... unfortunately our c++ implementation does not allow this.